app/Controllers/Http/SlackMatchController.js

```diff
@@ -0,0 +1,173 @@
+'use strict';
+
+const axios = require('axios')
+
+const Match = use('App/Models/Match');
+const User = use('App/Models/User');
+const Time = use('App/Models/Time');
+
+const { slackMessages, formatHour } = use('App/Controllers/utils.js');
+
+const { createChannelMessage, createErrorMessage, blockElements } = slackMessages;
+
+const matchListMessage = async () => {
+  const todayMatches = await Match.query()
+  .createdToday()
+  .withRelated()
+  .orderBy('time_id')
+  .fetch();
+
+  const text = blockElements.simpleRow('*--- Match list ---*');
+  
+  const matches = todayMatches.toJSON().map(match => {
+    const { time, user1, user2 } = match;
+    return user2 ?
+      blockElements.simpleRow(`${time.hour}: ${user1.name} vs ${user2.name}`) :
+      blockElements.rowWithButton(`${time.hour}: ${user1.name} waiting for Player 2...`, 'Play', `${match.id}`);
+  });
+
+  const blocks = [text, blockElements.dividerRow(), ...matches];
+  return createChannelMessage(blocks);
+}
+
+
+class SlackMatchController {
+  async matchStore ({ request, response }) {
+    try {
+      const { text, user_id } = request.body;
+      const parameters = text.split(' ');
+
+      if ((parameters.length < 1) || (parameters.length > 2)) {
```

Comment:
nitpicking, maybe we can use "includes" method to be more readable in this condition. I would suggest something like that 

```suggestion
      if (![1, 2].includes(parameters.length)) {
```

Reply:
great! that looks nice! Will update